front/src/lib/components/specialized/cookie-banner/detail-card.svelte

```diff
@@ -0,0 +1,104 @@
+<script lang="ts">
+  import RadioButtons from "$lib/components/inputs/radio-buttons.svelte";
+  import ArrowUpSLineArrows from "svelte-remix/ArrowUpSLineArrows.svelte";
+  import ArrowDownSLineArrows from "svelte-remix/ArrowDownSLineArrows.svelte";
+
+  let {
+    categoryConfig,
+    toggleConsentByKey,
+    disabled = false,
+    value = $bindable(),
+  } = $props();
+
+  let showCookieDetails = $state(false);
+  let detailsContainer: HTMLDivElement | undefined = $state();
+
+  if (categoryConfig.consentKey === "required") {
+    value = true;
+  }
+
+  $effect(() => {
+    if (showCookieDetails && detailsContainer) {
+      setTimeout(() => {
+        detailsContainer?.scrollIntoView({ behavior: "smooth", block: "end" });
+      }, 100);
+    }
+  });
+</script>
+
+{#snippet card(title, description, id, link = null)}
+  <div>
+    <div class="flex flex-col justify-between md:flex-row">
+      <div class="md:pr-s64 pr-0">
+        <h2 class="text-[1.25rem]">{title}</h2>
+        <p class="text-[1rem]">{description}</p>
```

Comment:
Tu n'as pas les warnings ?

<img width="1014" height="290" alt="Untitled" src="https://github.com/user-attachments/assets/8e86db64-1202-4803-a2a1-0a7b0d172067" />

```suggestion
        <h2 class="text-f20">{title}</h2>
        <p class="text-f16">{description}</p>
```

Reply:
Possible que ce soit l'extension _Tailwind CSS IntelliSense_ que j'ai installée.